nirc_ehr/resources/queries/study/deaths.js

```diff
@@ -9,6 +9,12 @@ function onInit(event, helper){
 
     helper.decodeExtraContextProperty('deathsInTransaction');
 
+    helper.setScriptOptions({
+        allowAnyId: true,
+        requiresStatusRecalc: true,
+        allowDatesInDistantPast: true
+    });
```

Comment:
Can you put these script options in nirc_triggers.js. That ensures it will override default EHR settings.

Reply:
moved